presto-orc/src/test/java/com/facebook/presto/orc/BenchmarkSelectiveStreamReaders.java

```diff
@@ -289,6 +296,10 @@ private final Object createValue()
                 return new SqlDate(random.nextInt());
             }
 
+            if (getType() == TIMESTAMP) {
+                return new SqlTimestamp(START_TIME.plus(random.nextLong()).getMillis(), UTC_KEY);
```

Comment:
Why not just `return new SqlTimestamp(random.nextLong());`?

Reply:
I need to pass the timeZone as the OrcTester writeLong is using getMillisUtc. I can get rid of the START_TIME though, used it to make it at least more than current time.
https://github.com/prestodb/presto/blob/6b05323bff91fb089b215737676819817da2befb/presto-orc/src/test/java/com/facebook/presto/orc/OrcTester.java#L947